erpnext/hr/doctype/vehicle_log/vehicle_log.js

```diff
@@ -25,6 +16,27 @@ frappe.ui.form.on("Vehicle Log", {
 		}
 	},
 
+	license_plate: function(frm){
+		if(frm.doc.license_plate){
+			frm.events.set_vehicle_details(frm)
+		}
+	},
+
+	set_vehicle_details: function(frm){
+		frappe.call({
+			method: "erpnext.hr.doctype.vehicle_log.vehicle_log.get_make_model",
+			args: {
+				license_plate: frm.doc.license_plate
+			},
+			callback: function(r) {
+				frappe.model.set_value(cur_frm.doctype, cur_frm.docname, "model", r.message[0]);
```

Comment:
Assign these values from server side as well in case vehicle log is created from API or server side and add a test case for this as well

Reply:
done